src/button-react.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import createButton from './button';
+
+export default class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.button = React.createRef();
+    this.componentDidMount = this.componentDidUpdate = () =>
+      createButton(this.button.current, this.props);
```

Comment:
While working on #38 I have found that the replacement of the DOM node breaks subsequent component updates. I have fixed it there by using a different API.